app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "

```

Comment:
Overall inserting non-parametrized data with f-string formatting is a bad practice. Here is no need to do it since you work with only one table and no need to get into the habit of doing this.
Also please follow SQL formatting style. Your statement should be rather broken down like so:
```
INSERT INTO actors (first_name, last_name) 
VALUES (?, ?)

Reply:
> Overall inserting non-parametrized data with f-string formatting is a bad practice. Here is no need to do it since you work with only one table and no need to get into the habit of doing this. Also please follow SQL formatting style. Your statement should be rather broken down like so:
> 
> ```
> INSERT INTO actors (first_name, last_name) 
> VALUES (?, ?)
> ```

Resolved